.github/CHANGELOG.md

```diff
@@ -268,6 +268,10 @@
   allowing complex matrices to be passed to `QubitUnitary`.
   [(#773)](https://github.com/PennyLaneAI/pennylane/pull/773)
 
+* Fixed a bug within `default.qubit`, resulting in greater efficiency
+  when applying a state vector to all wires on the device.
+  [(#849)](https://github.com/PennyLaneAI/pennylane/pull/849)
+
```

Comment:
What was the bug?

Reply:
Sorry, this is an update in master that does not reflect in the base branch. Will fix!